docusaurus.config.js

```diff
@@ -205,6 +205,26 @@ module.exports = {
                 },
               },
             ],
+            postBodyTags: [
+              {
+                tagName: 'script',
+                attributes: {
+                  type: 'text/javascript',
+                },
+                innerHTML: `
+                  document.addEventListener('DOMContentLoaded', function() {
+                    var copilot = document.createElement('co-pilot');
+                    copilot.setAttribute('copilot_id', 'your-copilot-id-here');
```

Comment:
Make sure you update `'your-copilot-id-here'` to your actual copilot id. Suggestion is to use env vars for this instead of hardcoding but that is up to you as the maintainers.